mantidproject/mantid
# Migrate v3.6.0 release notes to codebase

The release notes for v3.6.0 have been migrated to reStructuredText and integrated with the existing docs. As a result they will now be shipped with the code. The main purpose here is to have an illustrative set of examples for including release notes in future versions.

**To test:**
- Build the HTML documentation and view it locally in a web browser
- Check the navigation through from the main index page to each of the sub topics
- Check the formatting of each page is reasonable and all images have been included
- Check you agree with how the files have been named etc within the source code and resulting urls

The wiki version of the notes is [here](http://www.mantidproject.org/Release_Notes_3.6) for comparison.

Fixes #15434

Does not need to be in the release notes.

---
#### Reviewer

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):
##### Code Review
##### Functional Tests
- Does everything look good? Comment with the ship it emoji but don't merge. A member of `@mantidproject/gatekeepers` will take care of it.
